sftp/sftp.rst

```diff
@@ -231,35 +222,79 @@ A dedicated SFTP client application that has found favour in `our forum
 SFTP Access From WinSCP
 ^^^^^^^^^^^^^^^^^^^^^^^
 
-When you install WinSCP, it will ask you to pick your preferred interface either 
-the commander interface, or the explorer interface pictured below.  The WinSCP recommendation
-is for Windows users to chose the explorer interface.  If you have worked with ftp clients
-in the past, you may find the commander interface more familiar.  After you've installed
-WinSCP, when you start the program it will immediately bring up the Login window to start
-an SFTP session.
+When you install WinSCP, it will ask you to pick your preferred interface
+either the commander interface, or the explorer interface pictured below. The
+WinSCP recommendation is for Windows users to chose the explorer interface.If
```

Comment:
```suggestion
WinSCP recommendation is for Windows users to choose the explorer interface.If
```
@FroggyFlox Sorry but I've got another little suggestion/correction here.
We are recommending that they do this, rather than that they have already done it :).